src/components/ViewerSideNav/ViewerSideNav.tsx

```diff
@@ -24,27 +25,50 @@ import routes from '@/config/routes'
 
 type NavSubitem = {
   name: string
+  expandedName?: string
 }
 type NavItemType = {
   subitems?: NavSubitem[]
   icon: IconType
   to: string
 } & NavSubitem
 
-type SidenavProps = {
-  items: NavItemType[]
+export type SidenavProps = {
+  items?: NavItemType[]
+  isStudio?: boolean
 }
 
-const SideNavbar: React.FC<SidenavProps> = ({ items }) => {
-  const isStudio = useMatch(routes.studio())
+const SIDENAVBAR_ITEMS: NavItemType[] = [
+  {
+    icon: 'home-fill',
+    name: 'Home',
+    to: routes.index(),
+  },
+  {
+    icon: 'videos',
+    name: 'Videos',
+    to: routes.videos(),
+  },
+  {
+    icon: 'channels',
+    name: 'Channels',
+    to: routes.channels(),
+  },
+]
 
+const ViewerSideNavbar: React.FC<SidenavProps> = ({ items, isStudio }) => {
```

Comment:
This component is used for both viewer and studio so why is it called `ViewerSideNav`? We should have some generic `SideNav` component and then `StudioSideNav` and `ViewerSideNav` based on that. Or potentially we could just have the generic one and define the items in respective layouts, that could also work 

As a sidenote, maybe we also should use `Sidenav`, without the second capitalisation? Would be more in line with `Topbar`